src/Codeception/Test/Metadata.php

```diff
@@ -98,135 +95,92 @@ public function getCurrent($key = null)
         return $this->current;
     }
 
-    public function setCurrent(array $currents)
+    public function setCurrent(array $currents): void
     {
         $this->current = array_merge($this->current, $currents);
     }
 
-    /**
-     * @return mixed
-     */
     public function getName()
     {
         return $this->name;
     }
 
-    /**
-     * @param mixed $name
-     */
-    public function setName($name)
+    public function setName($name): void
     {
         $this->name = $name;
     }
 
-    /**
-     * @return mixed
-     */
     public function getFilename()
     {
         return $this->filename;
     }
 
-    /**
-     * @param mixed $index
-     */
-    public function setIndex($index)
+    public function setIndex($index): void
     {
         $this->index = $index;
     }
 
-    /**
-     * @return mixed
-     */
     public function getIndex()
     {
         return $this->index;
     }
 
-    /**
-     * @param mixed $filename
-     */
-    public function setFilename($filename)
+    public function setFilename($filename): void
     {
         $this->filename = $filename;
     }
 
-    /**
-     * @return array
-     */
-    public function getDependencies()
+    public function getDependencies(): array
     {
         return $this->params['depends'];
     }
 
-    public function isBlocked()
+    public function isBlocked(): bool
     {
         return $this->getSkip() !== null || $this->getIncomplete() !== null;
     }
 
-    /**
-     * @return mixed
-     */
     public function getFeature()
     {
         return $this->feature;
     }
 
-    /**
-     * @param mixed $feature
-     */
-    public function setFeature($feature)
+    public function setFeature($feature): void
```

Comment:
I'm pretty sure that $feature can only be string too.

Reply:
```shell
There was 1 error:

---------
1) TestLoaderTest: Load all tests
 Test  tests/unit/Codeception/TestLoaderTest.php:testLoadAllTests
                                                                                                                                                                                                                    
  [TypeError] Argument 1 passed to Codeception\Test\Metadata::setFeature() must be of the type string, null given, called in /home/runner/work/Codeception/Codeception/src/Codeception/Test/Gherkin.php on line 63  
                                                                                                                                                                                                                    
#1  /home/runner/work/Codeception/Codeception/tests/unit/Codeception/TestLoaderTest.php:68
#2  /home/runner/work/Codeception/Codeception/src/PHPUnit/Runner.php:117
#3  /home/runner/work/Codeception/Codeception/app.php:46
#4  /home/runner/work/Codeception/Codeception/app.php:47
#5  /home/runner/work/Codeception/Codeception/codecept:7

ERRORS!
Tests: 561, Assertions: 1329, Errors: 1, Skipped: 5.
```

```php
# Codeception\Test\Gherkin Line 63:

$this->getMetadata()->setFeature($featureNode->getTitle());
```

```php
# Behat\Gherkin\Node\FeatureNode Line 106-114:

/**
 * Returns feature title.
 *
 * @return null|string
 */
public function getTitle()
{
    return $this->title;
}
```

So I changed it to `?string`.